### PR TITLE
nfs-utils: new package at 2.4.3

### PR DESCRIPTION
**Pull request by djt3**
## Description of package
nfs-utils, to mount nfs shares

## Installed manifest of package

```
/var/lib/nfs/state
/var/lib/nfs/sm/
/var/lib/nfs/sm.bak/
/var/lib/nfs/rmtab
/var/lib/nfs/etab
/var/lib/nfs/
/var/lib/
/var/db/kiss/installed/nfs-utils/version
/var/db/kiss/installed/nfs-utils/sources
/var/db/kiss/installed/nfs-utils/manifest
/var/db/kiss/installed/nfs-utils/depends
/var/db/kiss/installed/nfs-utils/checksums
/var/db/kiss/installed/nfs-utils/build
/var/db/kiss/installed/nfs-utils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/umount.nfs.8
/usr/share/man/man8/statd.8
/usr/share/man/man8/sm-notify.8
/usr/share/man/man8/showmount.8
/usr/share/man/man8/rpcdebug.8
/usr/share/man/man8/rpc.statd.8
/usr/share/man/man8/rpc.sm-notify.8
/usr/share/man/man8/rpc.nfsd.8
/usr/share/man/man8/rpc.mountd.8
/usr/share/man/man8/nfsstat.8
/usr/share/man/man8/nfsiostat.8
/usr/share/man/man8/nfsd.8
/usr/share/man/man8/nfsconf.8
/usr/share/man/man8/mountstats.8
/usr/share/man/man8/mountd.8
/usr/share/man/man8/mount.nfs.8
/usr/share/man/man8/exportfs.8
/usr/share/man/man8/
/usr/share/man/man7/nfsd.7
/usr/share/man/man7/nfs.systemd.7
/usr/share/man/man7/
/usr/share/man/man5/nfsmount.conf.5
/usr/share/man/man5/nfs.conf.5
/usr/share/man/man5/nfs.5
/usr/share/man/man5/exports.5
/usr/share/man/man5/
/usr/share/man/
/usr/share/
/usr/
/sbin/umount.nfs4
/sbin/umount.nfs
/sbin/start-statd
/sbin/sm-notify
/sbin/showmount
/sbin/rpcdebug
/sbin/rpc.statd
/sbin/rpc.nfsd
/sbin/rpc.mountd
/sbin/nfsstat
/sbin/nfsiostat
/sbin/nfsconf
/sbin/mountstats
/sbin/mount.nfs4
/sbin/mount.nfs
/sbin/exportfs
/sbin/

```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
Binary directory should be `/usr/bin`.

**Comment by djt3** [comment]
new manifest, is this ok?:
```/var/lib/nfs/state
/var/lib/nfs/sm/
/var/lib/nfs/sm.bak/
/var/lib/nfs/rmtab
/var/lib/nfs/etab
/var/lib/nfs/
/var/lib/
/var/db/kiss/installed/nfs-utils/version
/var/db/kiss/installed/nfs-utils/sources
/var/db/kiss/installed/nfs-utils/manifest
/var/db/kiss/installed/nfs-utils/depends
/var/db/kiss/installed/nfs-utils/checksums
/var/db/kiss/installed/nfs-utils/build
/var/db/kiss/installed/nfs-utils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/umount.nfs.8
/usr/share/man/man8/statd.8
/usr/share/man/man8/sm-notify.8
/usr/share/man/man8/showmount.8
/usr/share/man/man8/rpcdebug.8
/usr/share/man/man8/rpc.statd.8
/usr/share/man/man8/rpc.sm-notify.8
/usr/share/man/man8/rpc.nfsd.8
/usr/share/man/man8/rpc.mountd.8
/usr/share/man/man8/nfsstat.8
/usr/share/man/man8/nfsiostat.8
/usr/share/man/man8/nfsd.8
/usr/share/man/man8/nfsconf.8
/usr/share/man/man8/mountstats.8
/usr/share/man/man8/mountd.8
/usr/share/man/man8/mount.nfs.8
/usr/share/man/man8/exportfs.8
/usr/share/man/man8/
/usr/share/man/man7/nfsd.7
/usr/share/man/man7/nfs.systemd.7
/usr/share/man/man7/
/usr/share/man/man5/nfsmount.conf.5
/usr/share/man/man5/nfs.conf.5
/usr/share/man/man5/nfs.5
/usr/share/man/man5/exports.5
/usr/share/man/man5/
/usr/share/man/
/usr/share/
/usr/sbin/start-statd
/usr/sbin/sm-notify
/usr/sbin/showmount
/usr/sbin/rpcdebug
/usr/sbin/rpc.statd
/usr/sbin/rpc.nfsd
/usr/sbin/rpc.mountd
/usr/sbin/nfsstat
/usr/sbin/nfsiostat
/usr/sbin/nfsconf
/usr/sbin/mountstats
/usr/sbin/exportfs
/usr/sbin/
/usr/
/sbin/umount.nfs4
/sbin/umount.nfs
/sbin/mount.nfs4
/sbin/mount.nfs
/sbin/
```

**Comment by dylanaraps** [comment]
No. All binaries should be in `/usr/bin/`.

**Comment by dylanaraps** [comment]
Tried doing `--sbindir=/usr/bin`?

**Comment by djt3** [comment]
--sbindir isnt needed since the prefix is specified as /usr. even with --sbindir set to /usr/sbin it leaves those last 4 in /sbin

**Comment by dylanaraps** [comment]
It should look like this: ```
/usr/bin/exportfs
/usr/bin/mount.nfs
/usr/bin/mount.nfs4
/usr/bin/mountstats
/usr/bin/nfsconf
/usr/bin/nfsiostat
/usr/bin/nfsstat
/usr/bin/rpc.mountd
/usr/bin/rpc.nfsd
/usr/bin/rpc.statd
/usr/bin/rpcdebugak
/usr/bin/showmount
/usr/bin/sm-notify
/usr/bin/start-statd
/usr/bin/umount.nfs
/usr/bin/umount.nfs4
/usr/bin/
/bin
```

I'll take a look and see if there's a way to fix this. :+1:

**Comment by djt3** [comment]
thanks :)

File: community/nfs-utils/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --disable-nfsv4 \
+    --disable-gss
+
```
Comment (dylanaraps) [inline community/nfs-utils/build:8]: Possibly try:

Possibly try (right before `make`):

```sh
sed -i 's|sbin|usr/bin|g' Makefile
sed -i "s|sbin|bin|" utils/nfsidmap/id_resolver.conf
```

Also add `--sbindir=/usr/bin` to configure. :+1:

Let me know what the manifest looks like afterwards.

**Comment by djt3** [comment]
new manifest:
/var/lib/nfs/state
/var/lib/nfs/sm/
/var/lib/nfs/sm.bak/
/var/lib/nfs/rmtab
/var/lib/nfs/etab
/var/lib/nfs/
/var/lib/
/var/db/kiss/installed/nfs-utils/version
/var/db/kiss/installed/nfs-utils/sources
/var/db/kiss/installed/nfs-utils/manifest
/var/db/kiss/installed/nfs-utils/depends
/var/db/kiss/installed/nfs-utils/checksums
/var/db/kiss/installed/nfs-utils/build
/var/db/kiss/installed/nfs-utils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/umount.nfs.8
/usr/share/man/man8/statd.8
/usr/share/man/man8/sm-notify.8
/usr/share/man/man8/showmount.8
/usr/share/man/man8/rpcdebug.8
/usr/share/man/man8/rpc.statd.8
/usr/share/man/man8/rpc.sm-notify.8
/usr/share/man/man8/rpc.nfsd.8
/usr/share/man/man8/rpc.mountd.8
/usr/share/man/man8/nfsstat.8
/usr/share/man/man8/nfsiostat.8
/usr/share/man/man8/nfsd.8
/usr/share/man/man8/nfsconf.8
/usr/share/man/man8/mountstats.8
/usr/share/man/man8/mountd.8
/usr/share/man/man8/mount.nfs.8
/usr/share/man/man8/exportfs.8
/usr/share/man/man8/
/usr/share/man/man7/nfsd.7
/usr/share/man/man7/nfs.systemd.7
/usr/share/man/man7/
/usr/share/man/man5/nfsmount.conf.5
/usr/share/man/man5/nfs.conf.5
/usr/share/man/man5/nfs.5
/usr/share/man/man5/exports.5
/usr/share/man/man5/
/usr/share/man/
/usr/share/
/usr/sbin/umount.nfs4
/usr/sbin/umount.nfs
/usr/sbin/start-statd
/usr/sbin/sm-notify
/usr/sbin/showmount
/usr/sbin/rpcdebug
/usr/sbin/rpc.statd
/usr/sbin/rpc.nfsd
/usr/sbin/rpc.mountd
/usr/sbin/nfsstat
/usr/sbin/nfsiostat
/usr/sbin/nfsconf
/usr/sbin/mountstats
/usr/sbin/mount.nfs4
/usr/sbin/mount.nfs
/usr/sbin/exportfs
/usr/sbin/
/usr/

**Comment by djt3** [comment]
or do you want the mount binaries in /usr/bin rather than /usr/sbin

**Comment by dylanaraps** [comment]
In `/usr/bin` always.

**Comment by dylanaraps** [comment]
Repology is listing `nfs-utils` as out of date (new version is 2.5.1)...

**Pull request merged**